packages/graphql/src/schemaTypes/objectTypes/gql-CurrentProject.ts

```diff
@@ -238,6 +239,25 @@ export const CurrentProject = objectType({
       description: 'If the browser is open or not',
       resolve: (source, args, ctx) => ctx.coreData.app.browserStatus,
     })
+
+    t.field('relevantRuns', {
+      type: RelevantRun,
+      description: 'Returns a list of relevant runs for matching Git shas from the Cloud',
+      resolve: async (source, args, ctx) => {
```

Comment:
Should this follow the pattern for `cloudSpec` and utilize the `remoteField`, allowing this resolver to be deferred? I don't think we want the frontend query to stall based on a cloud query.

Reply:
I do not need this anymore.  I am using a subscription to get this data since it needs to refresh with a poller